ngraph/core/src/op/experimental_detectron_prior_grid_generator.cpp

```diff
@@ -0,0 +1,157 @@
+//*****************************************************************************
+// Copyright 2017-2020 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
+
+#include <memory>
+
+#include "itt.hpp"
+#include "ngraph/attribute_visitor.hpp"
+#include "ngraph/op/experimental_detectron_prior_grid_generator.hpp"
+#include "ngraph/runtime/host_tensor.hpp"
+
+using namespace std;
+using namespace ngraph;
+
+NGRAPH_RTTI_DEFINITION(op::v6::ExperimentalDetectronPriorGridGenerator,
+                       "ExperimentalDetectronPriorGridGenerator",
+                       6);
+
+op::v6::ExperimentalDetectronPriorGridGenerator::ExperimentalDetectronPriorGridGenerator(
+    const Output<Node>& priors,
+    const Output<Node>& feature_map,
+    const Output<Node>& im_data,
+    const Attributes& attrs)
+    : Op({priors, feature_map, im_data})
+    , m_attrs(attrs)
+{
+    constructor_validate_and_infer_types();
+}
+
+bool op::v6::ExperimentalDetectronPriorGridGenerator::visit_attributes(AttributeVisitor& visitor)
+{
+    visitor.on_attribute("flatten", m_attrs.flatten);
+    visitor.on_attribute("h", m_attrs.h);
+    visitor.on_attribute("w", m_attrs.w);
+    visitor.on_attribute("stride_x", m_attrs.stride_x);
+    visitor.on_attribute("stride_y", m_attrs.stride_y);
+    return true;
+}
+
+shared_ptr<Node> op::v6::ExperimentalDetectronPriorGridGenerator::clone_with_new_inputs(
+    const OutputVector& new_args) const
+{
+    check_new_args_count(this, new_args);
```

Comment:
Please, add `NGRAPH_OP_SCOPE` macros like in the https://github.com/openvinotoolkit/openvino/pull/3730

Reply:
Done.